orchestration/scripts/check-existing-file.py

```diff
@@ -0,0 +1,26 @@
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from requests.exceptions import HTTPError
+import os
+
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+
+authed_session = AuthorizedSession(credentials)
+base_url = os.environ["API_URL"]
+dataset_id = os.environ["DATASET_ID"]
+target_path = os.environ["TARGET_PATH"]
+headers = {"accept": "application/json"}
+
+
+def check_file_existence(target_path: str):
+    response = authed_session.get(f"{base_url}/datasets/{dataset_id}/filesystem/objects",
+                                  params={"path": target_path},
+                                  headers=headers)
```

Comment:
This `headers` might be throwing things off; `AuthorizedSession` might not be overwriting them to inject the bearer token.

You shouldn't need the `accept` in practice, what happens if you remove this entirely?

Reply:
Not having the "accept" doesn't change anything, will remove it.